getting-started/kubernetes/vpp/baremetal.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Install VPP on a baremetal cluster
+description: Install Calico with the VPP dataplane on a baremetal cluster
+canonical_url: '/getting-started/kubernetes/vpp/baremetal'
+---
+
+### Requirements
+
+The following requirements are optional, but depending on your kernel version and hardware they may enable more efficient drivers to be used. To this end, ensure the following on all nodes:
```

Comment:
This is a bit fuzzy...can you provide "minimum" and "recommended" (that can enable more efficient drivers to be used)?

So, something like this:

**Minimum**

**Recommended**

**If using primary interface link**